logs/handler.go

```diff
@@ -66,19 +67,27 @@ func LogsHandler(c echo.Context) error {
 		})
 	}
 
-	modifiedForm := injectLabelsToForm(logSelector.Labels, form)
-	resp, err := makePostRequest(fmt.Sprintf("%s/%s", api.ApmHubPath, "search"), modifiedForm)
+	apmHubPayload := map[string]any{
+		"id":   reqData.ID,
```

Comment:
I think id should be component.external_id (then we don't need the kuberetes switch)   and type component.type, not logselector type.

Reply:
Done